README.md

```diff
@@ -107,6 +107,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | Project | Description |
 | --- | --- |
 | [GPflowOpt](https://github.com/GPflow/GPflowOpt)       | Bayesian Optimization using GPflow. |
+| [Trieste](https://github.com/secondmind-labs/trieste)       | Bayesian optimization with TensorFlow, with out-of-the-box support for GPflow models. |
```

Comment:
i put it with gpflowopt as it's a similar topic. Does that seem right?

Reply:
I'm happy for Trieste to be at the top together with GPflowOpt!